src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -0,0 +1,341 @@
+          <span slot="step-action">
+            <paper-button 
+                id="createServerButton" 
+                @tap="${this.handleProjectSetupNextTap}" 
+                ?disabled="${
+        !this.isProjectSetupNextEnabled(this.selectedProjectId, this.selectedBillingAccountId)}">
+              CREATE PROJECT
+            </paper-button>
+          </span>
+            <div class="section">
+              <div class="section-header">
+                <span class="stepcircle">1</span>
+                <div class="instructions">
+                  Name your new Google Cloud Project
+                </div>
+              </div>
+              <div class="section-content">
+                <paper-input value="${
+        this.selectedProjectId}" label="Project ID" always-float-label="" maxlength="100" @value-changed="${
+        this.onProjectIdChanged}"></paper-input>
+              </div>
+            </div>
+            
+            <div class="section">
+              <div class="section-header">
+                <span class="stepcircle">2</span>
+                <div class="instructions">
+                  Choose your preferred billing method for this project
+                </div>
+              </div>
+              <div class="section-content">
+                <paper-dropdown-menu no-label-float="">
+                  <paper-listbox slot="dropdown-content" selected="${
+        this.selectedBillingAccountId}" attr-for-selected="name" @selected-changed="${
+        this.onBillingAccountSelected}">
+                  ${this.billingAccounts.map(billingAccount => {
+      return html`<paper-item name="${billingAccount.id}">${billingAccount.name}</paper-item>`;
+    })}
+                  </paper-listbox>
+                </paper-dropdown-menu>
+              </div>
+            </div>
+        </outline-step-view>
+
+        <outline-region-picker-step id="regionPicker" .localize=${
+        this.localize} @region-selected="${this.onRegionSelected}">  
+        </outline-region-picker-step>
+      </iron-pages>`;
+  }
+
+  private async handleBillingVerificationNextTap(): Promise<void> {
+    this.billingAccounts = await this.account.listBillingAccounts();
+    if (!this.billingAccounts || this.billingAccounts.length === 0) {
+      // TODO: Show error
+    } else {
+      this.showProjectSetup();
+    }
+  }
+
+  private isProjectSetupNextEnabled(projectId: string, billingAccountId: string): boolean {
+    // TODO: Proper validation
+    return projectId !== '' && billingAccountId !== '';
+  }
+
+  private async handleProjectSetupNextTap(): Promise<void> {
+    this.project =
+        await this.account.createProject(this.selectedProjectId, this.selectedBillingAccountId);
+    this.showRegionPicker();
+  }
+
+  private showBillingAccountSetup(): void {
+    this.currentPage = 'billingAccountSetup';
+  }
+
+  private showProjectSetup(): void {
+    this.currentPage = 'projectSetup';
```

Comment:
You need to check the project state.
If the project is already created, you should set the project id accordingly.
If the billing is already set, you should select it.

This way we can handle failures in project setup up in a resilient way.